src/ClubInfo/Links.tsx

```diff
@@ -10,26 +12,59 @@ const LinkListDiv = styled.ul`
     margin-left: 0;
     margin-bottom: 10px;
   }
+  overflow: auto;
   margin-top: 20px;
   margin-left: 15%;
 `;
 
+const urlStart = /^https?:\/\/(www\.)?/;
+const urlQueries = /\?.*$/;
+
+function simplifyURL(url: string) {
+  return url.replace(urlStart, '').replace(urlQueries, '');
+}
+
+function getIcon(url: string): IconType {
+  const defaultIcon: IconType = FaLink;
+  const urlMap: { [key: string]: IconType } = {
+    'facebook.com': FaFacebook,
+    'instagram.com': FaInstagram,
+    'discord.gg': FaDiscord,
+  };
+
+  const domain = new URL(url).hostname.replace(/^www\./, '');
+
+  return urlMap[domain] ?? defaultIcon;
+}
+
 function LinkItem(props: { link: string }) {
   return (
-    <div>
-      <Icon icon={IoMdLink}></Icon>
-      <p>{props.link}</p>
-    </div>
+    <Button size="sm" style={{ maxWidth: '100%' }}>
```

Comment:
Is there a reason why this is a button / button styling?

Reply:
no good reason, i was having difficulty w/ CSS so this was an easy fix.

i replaced it with some CSS rules in 8454144 so this issue should be resolved